source/return_api.rst

```diff
@@ -255,9 +258,11 @@ total_value_delta              Yes         The delta value of transaction's gran
 payments                       No          Must be provided if original transaction has payments.
                                            Payments are list of :ref:`payment objects <Payment Object>`
 items                          Yes         Which items are returned. Items are list of :ref:`item objects<Item Object>`
-stamps_to_refund               No          Stamps payment to be returned
+stamps_to_add                  No          Stamps to be added manually. Can't be sent alongside stamps_to_deduct.
+stamps_to_deduct               No          Stamps to be deducted manually, if this field is present, automatic stamps calculation will be overriden by this.
```

Comment:
ini yang "automatic stamps calculation will be overriden" cuma untuk `stamps_to_deduct` kah?

Reply:
iya @hery733 , yang stamps_to_add gak ngubah transaction.stamps cuman add ke membership aja kayak award.